spec/Dockerfile.linuxdev

```diff
@@ -0,0 +1,90 @@
+FROM ubuntu:18.04
+
+ARG BOLT_PASSWORD=bolt
+ARG RUBY_VERSION=2.5.5
+# NOTE: Only designed to be set at build time, not at run time!
+ENV BOLT_PASSWORD=${BOLT_PASSWORD}
+ENV KRB5_REALM=
+ENV KRB5_KDC=
+ENV KRB5_ADMINSERVER=
+ENV SMB_ADMIN=Administrator
+ENV SMB_ADMIN_PASSWORD=
+
+# rbenv expects bash
+SHELL ["/bin/bash", "-c"]
+RUN useradd bolt \
+ && echo "bolt:${BOLT_PASSWORD}" | chpasswd
+
+# install Microsoft package repo for access to omi and powershell packages
+# gss-ntlmssp is for OMI server NTLM + Kerb support
+# ntp, realmd, sssd, samba-*, adcli are for working with Active Directory
+# krb5-config, krb5-user are Kerberos libraries
+# remaining packages are for .NET Core / PowerShell
+RUN export DEBIAN_FRONTEND=noninteractive \
+ && apt-get update -y \
+ && apt-get install -y wget \
+ && wget http://packages.microsoft.com/config/ubuntu/18.04/packages-microsoft-prod.deb \
+ && dpkg -i packages-microsoft-prod.deb \
+ && apt-get update -y \
+ && apt-get install -y \
+    vim \
+    git \
+    ssh \
+    dnsutils \
+    gettext-base \
+    gss-ntlmssp \
+    libgssapi-krb5-2 \
+    ntp \
+    krb5-config \
+    krb5-user \
+    # TODO: not sure which of these 3 are needed for SSH + Kerberos
```

Comment:
Aw boo I left a TODO in here... I was originally going to setup SSH to support Kerberos, but didn't get to doing that part. And I don't think this PR should be held up on that aspiration right now.

Reply:
Just to clarify on this - I'll remove the unneeded packages and the TODO from here prior to merge. I'll move that as a placeholder into the next PR as a reminder to either:

- investigate / add to that PR
- ditch for now